### PR TITLE
updated express bus times and coordinates

### PR DESCRIPTION
**Pull request by hannesmcman**
Added the new locations and times based on the schedule I found here: https://wp.stolaf.edu/sa/files/2012/08/Nfld-Express-2017.jpg

P.S. I'm Hannes and I want to get involved with All About Olaf, so I figured I'd start with something super easy! Thanks!

### CODE REVIEW

**Comment by hannesmcman** [comment]
Not sure why the CI build failed...do I need some sort of elevated permission for that to pass?

File: data/bus-times/express.yaml [hunk 1]
```diff
@@ -1,27 +1,26 @@
 line: Express Bus
 schedules:
   - days: [Mo, Tu, We, Th, Fr, Sa]
-    coordinates: [[44.462210, -93.183026], [44.460719, -93.155794], [44.454498, -93.161754], [44.433355, -93.186826], [44.448913, -93.171491], [44.454498, -93.161754], [44.460719, -93.155794], [44.462210, -93.183026]]
+    coordinates: [[44.462210, -93.183026], [44.460719, -93.155794], [44.454374, -93.162189], [44.438107, -93.182720], [44.432471, -93.187155], [44.435250, -93.188399], [44.441209, -93.181852], [44.449219, -93.171813], [44.454374, -93.162189], [44.460719, -93.155794], [44.462210, -93.183026]]
     stops:
-        [St. Olaf,  Carleton,  Food Co-op, Cub/Target, El Tequila, Food Co-op, Carleton,  St. Olaf]
+        [St. Olaf,  Carleton,  Food Co-op, Papa Murpheys, Cub/Target,      Arbys, Jesse James, El Tequila, Food Co-op,  Carleton,    St. Olaf]
```
Comment (rye) [inline data/bus-times/express.yaml]: Change `Papa Murpheys` to `Papa Murphy's` [EDIT: changed to `y's`, not `ys`.]? `Arbys` to `Arby's`? `Jesse James` to `Jesse James Lanes`?  Also, there are some strange spaces here—not necessarily anything wrong.  @hawkrives / @drewvolz, can you take a look at this whitespace?

File: data/bus-times/express.yaml [hunk 2]
```diff
@@ -1,27 +1,26 @@
...
+      - ['9:30pm',  '9:38pm',    '9:42pm',      '9:47pm',   '9:50pm',   '9:52pm',    '9:54pm',   '9:57pm',  '10:00pm', '10:03pm',   '10:10pm']
+      - ['10:15pm','10:23pm',   '10:27pm',     '10:32pm',  '10:35pm',  '10:37pm',   '10:39pm',  '10:42pm',  '10:45pm', '10:48pm',   '10:55pm']
 
   - days: [Su]
-    coordinates: [[44.462210, -93.183026], [44.460719, -93.155794], [44.454498, -93.161754], [44.433355, -93.186826], [44.448913, -93.171491], [44.454498, -93.161754], [44.460719, -93.155794], [44.462210, -93.183026]]
+    coordinates: [[44.462210, -93.183026], [44.460719, -93.155794], [44.454374, -93.162189], [44.438107, -93.182720], [44.432471, -93.187155], [44.435250, -93.188399], [44.441209, -93.181852], [44.449219, -93.171813], [44.454374, -93.162189], [44.460719, -93.155794], [44.462210, -93.183026]]
     stops:
-        [St. Olaf,  Carleton,  Food Co-op, Cub/Target, El Tequila, Food Co-op, Carleton,  St. Olaf]
+        [St. Olaf,  Carleton,  Food Co-op, Papa Murpheys, Cub/Target,      Arbys, Jesse James, El Tequila, Food Co-op,  Carleton,    St. Olaf]
```
Comment (rye) [inline data/bus-times/express.yaml]: Ditto here—change `Papa Murpheys` to `Papa Murphy's`? `Arbys` to `Arby's`? `Jesse James` to `Jesse James Lanes`?

**Review by rye — CHANGES_REQUESTED** [review]
Thanks for the good work, @hannesmcman!  Seems like they updated it again since the last time I updated this file.  (Which was #1539, my first Pull Request on this project, actually.)

Anyways, I just have a couple of tweaks to request, but it's totally possible we'll take it as-is.  Thanks again!

File: data/bus-times/express.yaml (hunk 1 above)
Comment (elijahverdoorn) [reply data/bus-times/express.yaml]: Thinking Murphys -> Murphy's (https://www.papamurphys.com/)

Comment (rye) [reply data/bus-times/express.yaml]: Yeah, agreed.  It's wrong on the bus schedule itself, which is why I'm curious, but we can always be better than that is.

**Comment by hannesmcman** [comment]
Good call on those names...just fixed them and added updated schedules for the red and blue lines as well. Also got rid of the weird spacing...I was trying to line the times up with the stop names, but it did look weird..

**Comment by hawkrives** [comment]
Sadly, I think the Express bus now has enough stops that it should switch to the expanded array notation (one per line, which is how the other two are formatted.)

@hannesmcman I probably accidentally broke the build for people who aren't admins, sorry! I'll look at fixing it, but we can merge this one as long as the JS build passes.

File: data/bus-times/express.yaml [hunk 3]
```diff
@@ -1,27 +1,26 @@
...
+      - ['8:45pm',  '8:53pm',  '8:57pm',  '9:02pm',  '9:05pm',  '9:07pm',  '9:09pm',  '9:12pm',  '9:15pm',  '9:18pm',   '9:25pm']
+      - ['9:30pm',  '9:38pm',  '9:42pm',  '9:47pm',  '9:50pm',  '9:52pm',  '9:54pm',  '9:57pm', '10:00pm', '10:03pm',  '10:10pm']
+      - ['10:15pm','10:23pm', '10:27pm', '10:32pm', '10:35pm', '10:37pm', '10:39pm', '10:42pm', '10:45pm', '10:48pm',  '10:55pm']
 
   - days: [Su]
-    coordinates: [[44.462210, -93.183026], [44.460719, -93.155794], [44.454498, -93.161754], [44.433355, -93.186826], [44.448913, -93.171491], [44.454498, -93.161754], [44.460719, -93.155794], [44.462210, -93.183026]]
+    coordinates: [[44.462210, -93.183026], [44.460719, -93.155794], [44.454374, -93.162189], [44.438107, -93.182720], [44.432471, -93.187155], [44.435250, -93.188399], [44.441209, -93.181852], [44.449219, -93.171813], [44.454374, -93.162189], [44.460719, -93.155794], [44.462210, -93.183026]]
     stops:
-        [St. Olaf,  Carleton,  Food Co-op, Cub/Target, El Tequila, Food Co-op, Carleton,  St. Olaf]
+        [St. Olaf,  Carleton,  Food Co-op, Papa Murpheys, Cub/Target, Arbys, Jesse James, El Tequila, Food Co-op,  Carleton,  St. Olaf]
```
Comment (hawkrives) [inline data/bus-times/express.yaml]: One more spot needs the apostrophes!

Comment (hawkrives) [reply data/bus-times/express.yaml]: Also, if you could reformat this into the one-stop-per-line format (Which is what the other two busses do) that'd be great.

File: data/bus-times/express.yaml [hunk 4]
```diff
@@ -1,27 +1,26 @@
 line: Express Bus
 schedules:
   - days: [Mo, Tu, We, Th, Fr, Sa]
-    coordinates: [[44.462210, -93.183026], [44.460719, -93.155794], [44.454498, -93.161754], [44.433355, -93.186826], [44.448913, -93.171491], [44.454498, -93.161754], [44.460719, -93.155794], [44.462210, -93.183026]]
+    coordinates: [[44.462210, -93.183026], [44.460719, -93.155794], [44.454374, -93.162189], [44.438107, -93.182720], [44.432471, -93.187155], [44.435250, -93.188399], [44.441209, -93.181852], [44.449219, -93.171813], [44.454374, -93.162189], [44.460719, -93.155794], [44.462210, -93.183026]]
     stops:
-        [St. Olaf,  Carleton,  Food Co-op, Cub/Target, El Tequila, Food Co-op, Carleton,  St. Olaf]
+        [St. Olaf,  Carleton,  Food Co-op, Papa Murphy's, Cub/Target, Arby's, Jesse James Lanes, El Tequila, Food Co-op, Carleton, St. Olaf]
```
Comment (hawkrives) [inline data/bus-times/express.yaml]: If you could reformat this into the one-stop-per-line format (Which is what the other two busses do) that'd be great.

File: data/bus-times/express.yaml [hunk 5]
```diff
@@ -1,27 +1,26 @@
...
+      - ['7:15pm',  '7:23pm',  '7:27pm',  '7:32pm',  '7:35pm',  '7:37pm',  '7:39pm',  '7:42pm',  '7:45pm',  '7:48pm',   '7:55pm']
+      - ['8:00pm',  '8:08pm',  '8:12pm',  '8:17pm',  '8:20pm',  '8:22pm',  '8:24pm',  '8:27pm',  '8:30pm',  '8:33pm',   '8:40pm']
+      - ['8:45pm',  '8:53pm',  '8:57pm',  '9:02pm',  '9:05pm',  '9:07pm',  '9:09pm',  '9:12pm',  '9:15pm',  '9:18pm',   '9:25pm']
+      - ['9:30pm',  '9:38pm',  '9:42pm',  '9:47pm',  '9:50pm',  '9:52pm',  '9:54pm',  '9:57pm', '10:00pm', '10:03pm',  '10:10pm']
+      - ['10:15pm','10:23pm', '10:27pm', '10:32pm', '10:35pm', '10:37pm', '10:39pm', '10:42pm', '10:45pm', '10:48pm',  '10:55pm']
 
   - days: [Su]
-    coordinates: [[44.462210, -93.183026], [44.460719, -93.155794], [44.454498, -93.161754], [44.433355, -93.186826], [44.448913, -93.171491], [44.454498, -93.161754], [44.460719, -93.155794], [44.462210, -93.183026]]
+    coordinates: [[44.462210, -93.183026], [44.460719, -93.155794], [44.454374, -93.162189], [44.438107, -93.182720], [44.432471, -93.187155], [44.435250, -93.188399], [44.441209, -93.181852], [44.449219, -93.171813], [44.454374, -93.162189], [44.460719, -93.155794], [44.462210, -93.183026]]
```
Comment (hawkrives) [inline data/bus-times/express.yaml]: And expand the Coordinates array, I guess

File: data/bus-times/express.yaml [hunk 6]
```diff
@@ -1,27 +1,26 @@
 line: Express Bus
 schedules:
   - days: [Mo, Tu, We, Th, Fr, Sa]
-    coordinates: [[44.462210, -93.183026], [44.460719, -93.155794], [44.454498, -93.161754], [44.433355, -93.186826], [44.448913, -93.171491], [44.454498, -93.161754], [44.460719, -93.155794], [44.462210, -93.183026]]
+    coordinates: [[44.462210, -93.183026], [44.460719, -93.155794], [44.454374, -93.162189], [44.438107, -93.182720], [44.432471, -93.187155], [44.435250, -93.188399], [44.441209, -93.181852], [44.449219, -93.171813], [44.454374, -93.162189], [44.460719, -93.155794], [44.462210, -93.183026]]
```
Comment (hawkrives) [inline data/bus-times/express.yaml]: And expand the Coordinates array, I guess

**Review by hawkrives — CHANGES_REQUESTED** [review]
Overall, a great first PR!

File: data/bus-times/express.yaml (hunk 4 above)
Comment (hannesmcman) [reply data/bus-times/express.yaml]: 351f29a should resolve this!

**Review by rye — APPROVED** [review]

**Review by elijahverdoorn — APPROVED** [review]
Good work! Thanks for your help!

**Review by hawkrives — APPROVED** [review]
LGTM!

**Pull request merged**